client/src/GlobalStyles.js

```diff
@@ -0,0 +1,58 @@
+import { createGlobalStyle } from "styled-components/macro";
+
+const GlobalStyle = createGlobalStyle`
+  *,
+  *::after,
+  *::before {
+    box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+        outline: 0;
+    }
+    ::placeholder {
+        color: #202020;
+        font-size: 1.3em;
+    }
+
+    :root {
+        font-size: 16px;
+    }
+    
+    body {
+        font-family: 'Roboto', sans-serif;
+        font-weight: 400;
+        color: #202020;
+        margin: 0;
+        text-rendering: optimizeLegibility !important;
+        -webkit-font-smoothing: antialiased !important;
+        -moz-osx-font-smoothing: grayscale !important;
+        user-select: none;
+        }
+    
```

Comment:
you shouldnt declare it in there
```suggestion
    :root {
        
    }
    
    body {
        font-family: 'Roboto', sans-serif;
        font-weight: 400;
        font-size: 16px;
        color: #202020;
        margin: 0;
        text-rendering: optimizeLegibility !important;
        -webkit-font-smoothing: antialiased !important;
        -moz-osx-font-smoothing: grayscale !important;
        user-select: none;
        }
    
```

Reply:
Thanks Alexis! 🧚🏻‍♂️